src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/QueryHelper.cs

```diff
@@ -31,7 +31,7 @@ public int GetOrganisationsTotalAmount()
                             .Type(Types.Parse("organisationdocument"))
                             .From(0)
                             .MatchAll());
-            return (int)results.HitsMetaData.Total;
+            return (results.HitsMetaData?.Total).HasValue ? 0 : (int)results.HitsMetaData.Total;
```

Comment:
The only thing I'd say is that the return statement is exactly the same in every method. Can this be refactored?

 `return (results.HitsMetaData?.Total).HasValue ? 0 : (int)results.HitsMetaData.Total;`

Reply:
I have no comments other than agree with Dave, a simple private method should suffice...